MapboxNavigation/NavigationMapView.swift

```diff
@@ -6,12 +6,13 @@ import Turf
 
 typealias CongestionSegment = ([CLLocationCoordinate2D], CongestionLevel)
 
-let routeLineWidthAtZoomLevels: [Int: MGLStyleValue<NSNumber>] = [
-    10: MGLStyleValue(rawValue: 8),
-    13: MGLStyleValue(rawValue: 9),
-    16: MGLStyleValue(rawValue: 11),
-    19: MGLStyleValue(rawValue: 22),
-    22: MGLStyleValue(rawValue: 28)
+
+let routeLineWidthAtZoomLevels: [Int: NSExpression] = [
```

Comment:
No need to wrap these constant values in NSExpression. The dictionary can be `[Int: Float]` or `[Int: Int]`, which also means you don’t need to declare the type, either.